src/Parsers/Access/ParserCreateUserQuery.cpp

```diff
@@ -16,8 +16,10 @@
 #include <base/range.h>
 #include <boost/algorithm/string/predicate.hpp>
 #include <base/insertAtEnd.h>
-#include <openssl/crypto.h>
-#include <openssl/rand.h>
+#if USE_SSL
+#     include <openssl/crypto.h>
+#     include <openssl/rand.h>
+#endif
```

Comment:
not sure it's a correct way to address this problem - there is probably something wrong with CI since we can compile it locally just fine... will try to figure it out